netpyne_ui/netpyne_geppetto.py

```diff
@@ -109,28 +109,18 @@ def importModel(self, modelParameters):
 
         os.chdir(owd)
     
-    def importCellTemplate(self, params):
+    def importCellTemplate(self, modelParameters):
         # Get Current dir
         owd = os.getcwd()
         
-        #Create Symbolic link
-        if params['compileMod']:
-            modPath = os.path.join(params['modFolder'],"x86_64")
-            subprocess.call(["rm", "-r", modPath])
-            os.chdir(params["modFolder"])
-            subprocess.call(["nrnivmodl"])
-        del params['compileMod']
-        
-        # Load mechanism if mod path is passed
-        if params['modFolder']:
-            neuron.load_mechanisms(str(params['modFolder'] ))
-        del params['modFolder']
-        
+        self.compileModMechFiles(modelParameters['compileMod'], modelParameters['modFolder'])
+
+        import netpyne_geppetto
```

Comment:
I'm not sure but I think this `import` will clear any previews definition. It works fine with `importModel` because there is nothing defined. But here we might delete previews defined populations. 

I think I tried that, but I'm not sure. Maybe you know.